src/VisualStudio/Core/Def/Implementation/Workspace/DetailedErrorInfoDialog.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<ui:DialogWindow x:Class="Microsoft.VisualStudio.LanguageServices.Implementation.DetailedErrorInfoDialog"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
+             xmlns:local="clr-namespace:Microsoft.VisualStudio.LanguageServices.Implementation"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:ui="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
+             xmlns:vs="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             mc:Ignorable="d"
+             ShowInTaskbar="False"
+             WindowStartupLocation="CenterOwner"
+             Background="{DynamicResource {x:Static vs:VsBrushes.ToolboxBackgroundKey}}"
+             Foreground="{DynamicResource {x:Static vs:VsBrushes.ToolboxGradientKey}}"
+             d:DesignHeight="300" d:DesignWidth="300"
+             SizeToContent="Height"
+             MaxWidth="768"
+             MinWidth="300"
+             MinHeight="300">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="*"/>
+            <RowDefinition Height="38"/>
+        </Grid.RowDefinitions>
+        <ScrollViewer Grid.Column="0" Grid.Row="0" HorizontalScrollBarVisibility="Auto" Style="{DynamicResource {x:Static vs:VsResourceKeys.ScrollViewerStyleKey}}">
+            <TextBox Name="stackTraceText" IsReadOnly="True"
```

Comment:
Why a text box inside of a ScrollViewer? Shouldn't the textbox just be multi-line?